### PR TITLE
Spotlight for CNCF DHHWG

### PR DESCRIPTION
**Pull request by sandeepkanabar**
### Description

This is a spotlight for the CNCF Deaf and hard-of-hearing working group (DHHWG)

### CODE REVIEW

**Comment by fsmunoz** [comment]
`contributor-site` PR: https://github.com/kubernetes/contributor-site/pull/530

File: content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## Call to Action
+**SK: Thank you for sharing your insights, Catherine. As we wrap up, do you have any final thoughts or a call to action for our readers?**
+
+**CP**: As we build our [accessibility page](https://contribute.cncf.io/accessibility/deaf-and-hard-of-hearing/), check in regularly to see what's new. Share the docs with your team, employer, and network — anyone, really. The more people understand what accessibility really means and why it matters, the more people will recognize when something isn't accessible, and be able to call out marketing-BS, which, unfortunately, is more often the case than not. We need allies to help push for change. **No minority can do this on their own**. So please learn about accessibility, keep an eye out for it, and call it out when something isn't accessible. We need your help! 
+
+# Wrapping up
```
Comment (fsmunoz) [inline content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md:74]:
```suggestion
## Wrapping up
```

File: content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+My first contribution was the [CNCF Landscape Guide](https://landscape.cncf.io/guide#introduction), which I co-authored with my former colleague Jason Morgan. Next, we started the [CNCF Glossary](https://glossary.cncf.io/), which explains cloud native concepts in simple terms. Today, the glossary has been (partially) localised into 14 languages! 
+
+Currently, I'm the co-chair of the [TAG Contributor Strategy](https://contribute.cncf.io/about/) and the Facilitator of the Deaf and Hard of Hearing Working Group (DHHWG) and Blind and Visually Impaired WG (BVIWG), which is still in formation. I'm also working on a new Linux Foundation (LF) initiative called ABIDE (Accessibility and Belonging through Inclusion, Diversity, and Equity), so stay tuned to learn more about it! 
+
+## Motivation and Early Milestones
```
Comment (fsmunoz) [inline content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md:20]:
```suggestion
## Motivation and early milestones
```

File: content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+![DHHWG members at KubeCon Chicago](cncf-dhhwg-chicago.jpg)
+*DHHWG members at KubeCon Chicago*
+
+The team has grown since then, and we've been able to do much more. With a kiosk in the project pavilion, an open space discussion, a sign language crash course, and a few media interviews, KubeCon Paris had a stronger advocacy and outreach focus. [Check out this video of our team in Paris](https://www.youtube.com/watch?v=E8AcyqsgAyQ) to get a glimpse of all the different KubeCon activities — it was such a great event! The team also launched the first CNCF Community Group in sign language, [Deaf in Cloud Native](https://community.cncf.io/deaf-in-cloud-native/), a glossary team that creates sign language videos for each technical term to help standardize technical signs across the globe. It's crazy to think that it all happened within one year! 
+
+## Overcoming Challenges and Addressing Misconceptions
```
Comment (fsmunoz) [inline content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md:33]:
```suggestion
## Overcoming challenges and addressing misconceptions
```

File: content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+**SK: Are there any misconceptions about the deaf and hard-of-hearing community in tech that you'd like to address?**
+
+**CP**: Deaf and hard of hearing individuals are very diverse — there is no one-size-fits-all. Some deaf people are oral (speak), others sign, while some lip read or prefer captions. It generally depends on how people grew up. While some people come from deaf families and sign language is their native language, others were born into hearing families who may or may not have learned how to sign. Some deaf people grew up surrounded by hearing people, while others grew up deeply embedded in Deaf culture. Hard-of-hearing individuals, on the other hand, typically can communicate well with hearing peers one-on-one in quiet settings, but loud environments or conversations with multiple people can make it hard to follow the conversation. Most rely heavily on captions. Each background and experience will shape their communication style and preferences. In short, what works for one person, doesn't necessarily work for others. So NEVER ASSUME and ALWAYS ASK about accessibility needs and preferences.
+
+## Impact and the Role of Allies
```
Comment (fsmunoz) [inline content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md:42]:
```suggestion
## Impact and the role of allies
```

File: content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+Interpreters are most valuable during networking, not in talks (with captions). Most people come to conferences for the hallway track. That is no different for deaf attendees. If they can't network, they are missing out on key professional connections, affecting their career prospects. 
+
+
+**SK: In your view, how crucial is the role of allies within the DHHWG, and what contributions have they made to the group’s success?**
+
+**CP**: Deaf and hard of hearing individuals are a minority and can only do so much. ***Allies are key to any diversity and inclusion initiative***. As a majority, allies can help spread the word and educate their peers, playing a key role in scaling advocacy efforts. They also have the power to demand change. It's easy for companies to ignore minorities, but if the majority demands that their employers be accessible, environmentally conscious, and good citizens, they will ultimately be pushed to adapt to new societal values. 
+
+## Expanding DEI Efforts and Future Vision
```
Comment (fsmunoz) [inline content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md:54]:
```suggestion
## Expanding DEI efforts and future vision
```

**Review by fsmunoz — CHANGES_REQUESTED** [review]
We follow the Docs style guide and do not capitalise headings.

**Review by sftim — COMMENTED** [review]
This might be a better fit for the CNCF blog

/hold
pending confirmation that the Kubernetes blog is the right home for this article.

**Comment by sandeepkanabar** [comment]
> This might be a better fit for the CNCF blog
> 
> /hold pending confirmation that the Kubernetes blog is the right home for this article.

Understand Tim @sftim. The aim here is to drive inclusion among contributors and kubernetes folks so that they can be allies to the DHHWG and can help mentor folks from the WG. The CNCF will also publish a blog on the WG this Deaf Awareness Month, in addition to a host of other activities too. While there is an audience overlap, publishing on both blogs will allow us to reach a much bigger audience.

**Comment by fsmunoz** [comment]
@sftim I agree that this is a corner-case, since it's a CNCF Working Group. The reasons why I think that it fits within k8s.dev (at least), and to add to the discussion, re:

* The link to the Kubernetes ecosystem is the starting point of the conversation, and explicitly mentioned and discussed.
* This WG has presented in KubeCons (that's the first time I heard them), and the subject matter is one which doesn't fit a k8s WG, while at the same time being important to k8s.
* It specifically addresses how to make KubeCon itself more accessible.

This doesn't mean it wouldn't also be suitable for the CNCF blog (although I'm not knowledgeable about the publication process or criteria there), just that I personally see it with sufficient contact area with the Kubernetes community. I do however understand that these might be seen as insufficient for others.

**Comment by sftim** [comment]
> This WG has presented in KubeCons (that's the first time I heard them), and the subject matter is one which doesn't fit a k8s WG, while at the same time being important to k8s.

That conference, _KubeCon + CloudNativeCon_, is not just about Kubernetes.

We should leave the hold in place until this is settled.

**Comment by sftim** [comment]
(this would also need an assigned publication date)

**Comment by sandeepkanabar** [comment]
> (this would also need an assigned publication date)

The aim was to get it published around Sep 24. Sep 23 is the International Day of Sign Languages (IDSL) and we wanted to publish this around the date. Also, September is being celebrated as Deaf Awareness month too.

**Comment by sftim** [comment]
If aiming for publication in the next 7 days I strongly recommend not targeting https://k8s.io/blog/ - we are rather short on review capacity, and I personally like the idea of ensuring we do good quality reviews on every article we publish there.

There's no formal SLA, but I've been suggesting a 4 week turnaround time. We also hope to encourage more people to contribute as blog reviewers and editors.

The audience for https://k8s.dev/blog/ is smaller and we can worry less about getting our message exactly right.

File: content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+## Expanding DEI efforts and future vision
+**SK: The importance of allies in driving change is clear. Beyond the DHHWG, are you involved in any other DEI groups or initiatives within the tech community?**
+
+**CP**: As mentioned above, I'm working on an initiative called ABIDE, which is still work in progress. I don't want to share too much about it yet, but what I can say is that the DHHWG will be part of it and that we just started a Blind and Visually Impaired WG (BVIWG). ABIDE will start by focusing on accessibility, so if anyone reading this has an idea for another WG, please reach out to me via the CNCF Slack @Catherine Paganini.
+
+**SK: What does the future hold for the DHHWG? Can you share details about any ongoing or upcoming initiatives?**
+
+**CP**: I think we've been very successful in terms of visibility and awareness so far. We can't stop, though. Awareness work is ongoing, and most people in our community haven't heard about us or met anyone on our team yet, so a lot of work still lies ahead. 
+![DHHWG members at KubeCon Paris](cncf-dhhwg-paris.jpg)
```
Comment (stmcginnis) [inline content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md:62]: Line breaks are needed before and after the image link so this doesn't get rendered all together.

**Review by stmcginnis — COMMENTED** [review]
One minor rendering issue. Other than that, I didn't notice any typos or grammar issues. Looks great!

File: content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md (hunk 6 above)
Comment (sandeepkanabar) [reply content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md:64]: Thanks @stmcginnis . Incorporated your suggestion and added line breaks before and after both the image links.

**Comment by sandeepkanabar** [comment]
@sftim - I've a question - We have kept publish date as 30th. So will that be UTC of 30th, or PST or 30th?

**Pull request merged**